wiki/Installation/macOS/tr.md

```diff
@@ -0,0 +1,20 @@
+# macOS için kurulum
+
+Bu sayfa size macOS cihazınızda osu!'yu nasıl çalıştıracağınızı kabaca açıklayacaktır.
+
+## Minimum sistem gereksinimleri
+
+- macOS çalıştıran ortalama bir bilgisayar.
```

Comment:
`yeterince iyi`? `düzgün`? - `decent` probably shouldn't be translated as `ortalama` 

funny how osu wiki be like that though

Reply:
meh, "decent" here can perfectly be translated as "ortalama" here. another possibility would be translating it as "iyi" which I used in the original Installation article